Threads/src/main/java/task1/Operator.java

```diff
@@ -0,0 +1,44 @@
+package task1;
+
+import static java.lang.Thread.sleep;
+
+public class Operator implements Runnable{
+
+    private int countCards = 25;
+    private int countGuests = 100;
+    private Card card;
+
+    public Operator(Card _card)
+    {
+        this.card = _card;
```

Comment:
Naming Conventions:
Variable names should not start with underscore _ or dollar sign $ characters